x-pack/platform/plugins/shared/fleet/server/tasks/sync_integrations/compare_synced_integrations.ts

```diff
@@ -332,3 +338,61 @@ export const getRemoteSyncedIntegrationsStatus = async (
     return { error, integrations: [] };
   }
 };
+
+export const getRemoteSyncedIntegrationsInfoByOutputId = async (
```

Comment:
should we move this to a separate file?

Reply:
I just thought it's very related to the above function, but yes we can move it if you think it's more readable.

Reply:
Done!